packages/playground/src/index.js

```diff
@@ -3,6 +3,7 @@ import { Theme as MuiTheme } from "@rjsf/material-ui";
 import { Theme as FluentUITheme } from "@rjsf/fluent-ui";
 import { Theme as SuiTheme } from "@rjsf/semantic-ui";
 import { Theme as AntdTheme } from "@rjsf/antd";
+import { Theme as Bootstrap4Theme } from "../../bootstrap-4";
```

Comment:
```suggestion
import { Theme as Bootstrap4Theme } from "@rjsf/bootstrap-4";
```

Reply:
Like as of now it is not published right, so for the CI to work fine. we are using this. Once CI is passed, we can change it in this PR or in a separate PR

Reply:
If you run `npm install` from the root directory, lerna will symlink `@rjsf/bootstrap-4` to point to the local directory. It doesn't need to be published.

Reply:
updated